projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -655,7 +655,7 @@ export class IgxOverlayService implements OnDestroy {
                     // if we should exclude position target check if the click is over it. If so do not close overlay
                     const positionTarget = info.settings.target as HTMLElement;
                     let clickOnPositionTarget = false;
-                    if (positionTarget) {
+                    if (positionTarget && positionTarget.contains) {
```

Comment:
Since this is pretty much the only use of the target here, you could also get away with a type check which will also let you not hard type assert `as HTMLElement` above too.

PS: I realize this is not cross-frame friendly, but then again there's a precedent of `instanceof HTMLElement` check in utils and none of the positioning or these handlers support targets in other frames atm, so those rare cases will have to resort to point use anyway.